examples/build.gradle

```diff
@@ -0,0 +1,33 @@
+project.ext.sdkVersion = 23
+project.ext.buildTools = '23.0.0'
+
+allprojects {
+    def currentVersion = new File("${rootDir}/../version.txt").text.trim()
+
+    buildscript {
+        repositories {
+            //mavenLocal()
+            maven {
+                url "${System.env.ANDROID_HOME}/extras/android/m2repository"
+            }
+            jcenter()
+        }
+        dependencies {
+            classpath 'com.android.tools.build:gradle:1.2.3'
+            classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2'
+            classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
+            classpath 'com.novoda:gradle-android-command-plugin:1.3.0'
+            classpath "io.realm:gradle-plugin:${currentVersion}"
+        }
+    }
+
+    group = 'io.realm'
+    version = currentVersion
+
+    repositories {
+        maven {
+            url "${System.env.ANDROID_HOME}/extras/android/m2repository"
```

Comment:
this needs "local.proeprties" to be set, can't we just use `${android.sdkDirectory}` ?